CHANGELOG.md

```diff
@@ -1,5 +1,126 @@
+* Local storage is now cleaned appropriately after the test run.([#1546](https://github.com/DevExpress/testcafe/issues/1546))
+* TestCafe now checks element visibility with a timeout when the target element's `style.top` is negative ([#1185](https://github.com/DevExpress/testcafe/issues/1185))
+* Fetching an absolute CORS URL now works correctly. ([#1629](https://github.com/DevExpress/testcafe/issues/1629))
+* Fixed a bug that led to an error in test code. ([#1442](https://github.com/DevExpress/testcafe/issues/1442))
+* TypeScript performance has been enhanced. ([#1591](https://github.com/DevExpress/testcafe/issues/1591))
+* Mousedown on a link without `src` is now handled properly. ([#1598](https://github.com/DevExpress/testcafe/issues/1598))
+* Fixed a bug that led to test failing in Edge 15. ([testcafe-hammerhead/#1176](https://github.com/DevExpress/testcafe-hammerhead/issues/1176))
+* The right port is now applied to a cross-domain iframe location after redirect. ([testcafe-hammerhead/#1191](https://github.com/DevExpress/testcafe-hammerhead/issues/1191))
+* `Hammerhead.EVENTS.xhrSend` and `Hammerhead.EVENTS.fetchSend` are renamed to `...Sent`. ([testcafe-hammerhead/#678](https://github.com/DevExpress/testcafe-hammerhead/issues/678))
+* All internal properties are marked as non-enumerable. ([testcafe-hammerhead/#1182](https://github.com/DevExpress/testcafe-hammerhead/issues/1182))
+* Hammerhead no longer crashes on the `slack.com/signin` page. ([testcafe-hammerhead/#1195](https://github.com/DevExpress/testcafe-hammerhead/issues/1195))
+* Fixed a bug where a script error occurred after proxying. ([testcafe-hammerhead/#1209](https://github.com/DevExpress/testcafe-hammerhead/issues/1209))
+* WebWorker content is now correctly proxied in FireFox 54. ([testcafe-hammerhead/#1216](https://github.com/DevExpress/testcafe-hammerhead/issues/1216))
+* Methods that return live node collections are now correctly proxied. ([testcafe-hammerhead/#1096](https://github.com/DevExpress/testcafe-hammerhead/issues/1096))
+* Code instrumentation for the `document.activeElement` property works properly if it is `null`. ([testcafe-hammerhead/#1226](https://github.com/DevExpress/testcafe-hammerhead/issues/1226))
+* `length`, `item` and `namedItem` are no longer own properties of `LiveNodeListWrapper`. ([testcafe-hammerhead/#1222](https://github.com/DevExpress/testcafe-hammerhead/issues/1222))
+* The `scope` option in the `serviceWorker.register` function is now processed correctly. ([testcafe-hammerhead/#1233](https://github.com/DevExpress/testcafe-hammerhead/issues/1233))
+* Promises from a fetch request are now processed correctly. ([testcafe-hammerhead/#1234](https://github.com/DevExpress/testcafe-hammerhead/issues/1234))
+* Fixed a bug that led to an error in compiled babel code. ([testcafe-hammerhead/#1231](https://github.com/DevExpress/testcafe-hammerhead/issues/1231))
```

Comment:
Change to some meaningful title (as I see it a problem with for...of) /cc @miherlosev